upgrade/src/migrateToMCD.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config()
+const Web3 = require('web3')
+const multiSigWalletAbi = require('../abi/multiSigwallet')
+const proxyAbi = require('../../build/contracts/EternalStorageProxy').abi
+const foreignBridgeAbi = require('../../build/contracts/ForeignBridgeErcToNative').abi
+const confirmTransaction = require('./confirmTransaction')
+
+const {
+  FOREIGN_PRIVKEY,
+  FOREIGN_RPC_URL,
+  FOREING_BRIDGE_ADDRESS,
+  ROLE,
+  FOREIGN_START_BLOCK,
+  FOREIGN_GAS_PRICE
+} = process.env
+
+const web3 = new Web3(new Web3.providers.HttpProvider(FOREIGN_RPC_URL))
+const { address } = web3.eth.accounts.wallet.add(FOREIGN_PRIVKEY)
+
+const migrateToMCD = async () => {
+  try {
+    const proxy = new web3.eth.Contract(proxyAbi, FOREING_BRIDGE_ADDRESS)
+    const bridge = new web3.eth.Contract(foreignBridgeAbi, FOREING_BRIDGE_ADDRESS)
+
+    const ownerAddress = await proxy.methods.upgradeabilityOwner().call()
+    const multiSigWallet = new web3.eth.Contract(multiSigWalletAbi, ownerAddress)
```

Comment:
is it worth to check that the account that is going to send a transaction participates in the multisig wallet?

Reply:
Added the file `upgrade/src/validatorState.js` that check that the validator is part of the multisig wallet